src/main/java/games/strategy/triplea/ui/AbstractMovePanel.java

```diff
@@ -244,7 +245,7 @@ protected final void display(final PlayerID id, final String actionLabel) {
       this.actionLabel.setText(id.getName() + actionLabel);
       add(leftBox(this.actionLabel));
       if (setCancelButton()) {
-        add(leftBox(new JButton(SwingComponents.newAbstractAction(this::cancelMove))));
+        add(leftBox(new JButton(cancelMoveAction)));
```

Comment:
Would it work to store a `JButton` instead of an `AbstractAction`. That avoids the redundancy of specifying the button title both as part of the button and again as part of the abstract action, and would avoid new'ing up a JButton here.